vignettes/git_usage.Rmd

```diff
@@ -175,4 +175,53 @@ knitr::include_graphics("github_conflicts.png", dpi = 144)
 * [GitHub and RStudio](https://resources.github.com/whitepapers/github-and-rstudio/)
 * [Happy Git and GitHub for the useR](https://happygitwithr.com/)
 
+# Package Release
+
+## Quarterly Release
+
+A package release is done in four parts:
+
+1) Declare a freeze on all Pull Requests going into the `devel` branch via internal channels.
+1) Create a PR from `devel` into the `pre-release` branch.  Verify that all CI/CD checks are passing and bundle up and send off to [CRAN](https://cran.r-project.org/).
+1) Once package is available on CRAN, another Pull Request is created for merging the `pre-release` branch the `main` branch.  This will trigger the action to rebuild the `{admiral}` website with all the updates for this release.
+1) Use the release button on GitHub to "release" the package onto GitHub.  This release onto Github archives the version of code within the `main` branch, attaches the News/Changelog file, bundles the code into a `tar.gz` file and makes a validation report via the GitHub action `validation` from [insightsengineering/validatoR](https://github.com/insightsengineering/thevalidatoR).  Please see past [admiral releases](https://github.com/pharmaverse/admiral/releases) for reference. 
+
+**Quarterly Release:**  `devel >> pre-release >> main` 
+
+## Hot Fix Release
+
+Occasionally we will need to release a hot fix to address a package breaking bug.  A hot fix release is done in 6 parts:
+
+1) Create two branches called `patch` and `pre-release` off the current `main`.
+1) Pull Requests that address the bugs should be merged into the `patch` branch.
+1) Create a PR from `patch` into a newly created `pre-release` branch off of `main`.  Verify that all CI/CD checks are passing and bundle up and send off to [CRAN](https://cran.r-project.org/).
+1) Once package is approved and available on CRAN, another PR is created for merging the `pre-release` branch into the `main` branch.  This will trigger the action to rebuild the `{admiral}` website with all the updates for this hot fix release.
+1) Use the release button on GitHub to "release" the package onto GitHub.  This release onto Github archives the version of code within the `main` branch, attaches the News/Changelog file, bundles the code into a `tar.gz` file and makes a validation report via the GitHub action `validation` from [insightsengineering/validatoR](https://github.com/insightsengineering/thevalidatoR).  Please see past [admiral releases](https://github.com/pharmaverse/admiral/releases) for reference.
+1) These hot fixes should then be merged into the `devel` branch through an additional PR.  
+
+**Hot Fix Release**: `patch >> pre-release >> main  >> devel`
+
+##  Release Schedule
```

Comment:
why repeat this section from the README? you'll have to update it in 2 places then. i would just leave it on the README only

Reply:
I kind of like it being here.  I will be make sure its gets updated - its on the release checklist when updating the description and changelog files - so i don't forget

Reply:
@bms63 couldn't it just be linked back to the main page from the README via a link here? sorry but i have a real bugbear about exactly the same info being maintained in 2 places. @thomas-neitmann any thoughts??

Reply:
i can just do a link.  np at all

Reply:
link provided, but it won't go to the updated readme until the website is re-depolyed.  I couldn't figure out where the `.html` exists for the readme in the sitemap..  

@dinakar29 do you know how I can link to the updated readme on the pkgdown site when I create a local version for the release schedule.

We can link do different pages and sections using e.g. (programming_strategy.html#deprecation-1), but this doesn't works for (readme.html#release-schedule).

Reply:
Maybe `../index.html#release-schedule`?

Reply:
worked like a charm!